concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+Zhou-2020-350-181	181	sound	聲音	2879	SOUND (OF INSTRUMENT OR VOICE)
+Zhou-2020-350-182	182	sour	酸	1906	SOUR
+Zhou-2020-350-183	183	soy bean	豆	2027	SOYA
+Zhou-2020-350-184	184	sparrow	喜鵲	1854	SPARROW
+Zhou-2020-350-185	185	spider	蜘蛛	843	SPIDER
+Zhou-2020-350-186	186	stone, bone	石頭、骨頭	857	STONE
+Zhou-2020-350-187	187	straight	直	1404	STRAIGHT
+Zhou-2020-350-188	188	sun	太陽	1343	SUN
+Zhou-2020-350-189	189	swollen	腫	1573	SWELL
+Zhou-2020-350-190	190	tael	兩		
+Zhou-2020-350-191	191	ten	十	1515	TEN
+Zhou-2020-350-192	192	thick	厚	1244	THICK
+Zhou-2020-350-193	193	thick (liquid)	濃	1244	THICK
+Zhou-2020-350-194	194	thin (<> thick)	薄		
+Zhou-2020-350-195	195	thorn	刺兒	124	THORN
+Zhou-2020-350-196	196	thread	綫	1161	THREAD
+Zhou-2020-350-197	197	three	三	492	THREE
+Zhou-2020-350-198	198	thunder, to sound	雷、響		
+Zhou-2020-350-199	199	tiger	老虎	846	TIGER
+Zhou-2020-350-200	200	to answer	答應	1275	ANSWER
+Zhou-2020-350-201	201	to arrive	到	1387	ARRIVE
+Zhou-2020-350-202	202	to ask, to tell	問、告訴	1309	ASK (INQUIRE)
+Zhou-2020-350-203	203	to be able to	會	972	BE ABLE
+Zhou-2020-350-204	204	to be willing to	肯		
+Zhou-2020-350-205	205	to become	變成	1990	BECOME
+Zhou-2020-350-206	206	to beg	乞討	3534	BEG
+Zhou-2020-350-207	207	to belong to a Chinese zodiac	屬（豬）		
+Zhou-2020-350-208	208	to bite	咬、吃	1403	BITE
+Zhou-2020-350-209	209	to blink	眨	1655	BLINK
+Zhou-2020-350-210	210	to block	堵塞	2762	BLOCK (THE WAY)
+Zhou-2020-350-211	211	to blow	吹	175	BLOW (OF WIND)
+Zhou-2020-350-212	212	to boil	煮	2492	BOIL (SOMETHING)
+Zhou-2020-350-213	213	to break (from the middle)	斷	3020	BREAK (BREAKING)
+Zhou-2020-350-214	214	to broke in the middle	破、斷		
+Zhou-2020-350-215	215	to burn	燃燒	2102	BURN
+Zhou-2020-350-216	216	to burn	燒	2102	BURN
+Zhou-2020-350-217	217	to carry	搬	700	CARRY
+Zhou-2020-350-218	218	to carry on the back	背 v.	465	CARRY ON BACK
+Zhou-2020-350-219	219	to chisel	鑿	1979	CHISEL
+Zhou-2020-350-220	220	to chop	剁	155	CHOP
+Zhou-2020-350-221	221	to comb	梳	2070	COMB (VERB)
+Zhou-2020-350-222	222	to compel	逼		
+Zhou-2020-350-223	223	to connect	連接	2862	CONNECT
+Zhou-2020-350-224	224	to contain	裝	697	CONTAIN
+Zhou-2020-350-225	225	to cough	咳嗽	879	COUGH
+Zhou-2020-350-226	226	to cover up	蓋上	1719	BURY
+Zhou-2020-350-227	227	to crawl	爬、上	96	CRAWL
+Zhou-2020-350-228	228	to crow	啼	2074	CROW (VERB)
+Zhou-2020-350-229	229	to cry	哭	1839	CRY
+Zhou-2020-350-230	230	to cut with a saw	鋸		
+Zhou-2020-350-231	231	to cut with knife	割		
+Zhou-2020-350-232	232	to die	死	1494	DIE
+Zhou-2020-350-233	233	to do	做	813	DO
+Zhou-2020-350-234	234	to dream	夢	1920	DREAM (SOMETHING)
+Zhou-2020-350-235	235	to drip	撒、滴	808	DRIP (FALL IN GLOBULES)
+Zhou-2020-350-236	236	to drive away	趕	30	DISPEL
+Zhou-2020-350-237	237	to eat	吃	1336	EAT
+Zhou-2020-350-238	238	to escape	逃	1615	ESCAPE
+Zhou-2020-350-239	239	to exchange	交換	2509	EXCHANGE
+Zhou-2020-350-240	240	to exit	出	1107	GO OUT
+Zhou-2020-350-241	241	to expose under sun	曬		
+Zhou-2020-350-242	242	to fear	怕	1419	FEAR (BE AFRAID)
+Zhou-2020-350-243	243	to feed	餵	739	FEED
```

Comment:
I couldn't find this or the gloss below. @LinguList Can both of them be mapped to FEED?

Reply:
@LinguList What do you think?

Reply:
This means "to feed".